entries/s/seznam.cz.json

```diff
@@ -2,13 +2,15 @@
   "Seznam.cz": {
     "domain": "seznam.cz",
     "tfa": [
+      "totp",
       "custom-software"
     ],
     "custom-software": [
       "Seznam.cz mobile app"
     ],
-    "documentation": "https://blog.seznam.cz/2018/04/seznam-cz-ma-nove-dvoufazove-overeni-u-vsech-svych-sluzeb-zvysi-zabezpeci-prihlasovani-napriklad-do-emailu/",
+    "documentation": "https://napoveda.seznam.cz/cz/login/dvoufazove-overeni/",
     "categories": [
+      "other",
```

Comment:
The `other` category is only meant for sites that can't be put in any other category, and as Seznam.cz already fits into one category, I'm afraid it doesn't qualify for the `other` category.
```suggestion
```

Reply:
I see, misunderstood the category, sorry. I've removed the commit adding the category.